cert_manager/metadata.csv

```diff
@@ -2,8 +2,8 @@ metric_name,metric_type,interval,unit_name,per_unit_name,description,orientation
 cert_manager.prometheus.health,gauge,,,,Whether the check is able to connect to the metrics endpoint,0,cert-manager,cm.health
 cert_manager.certificate.ready_status,gauge,,,,The ready status of the certificate,0,cert-manager,cm.cert_ready_status
 cert_manager.certificate.expiration_timestamp,gauge,,second,,The date after which the certificate expires. Expressed as a Unix Epoch Time,0,cert-manager,cm.cert_exp_time
-cert_manager.acme_client.request.count,count,,,,The number of requests made by the ACME client,1,cert-manager,cm.acme_req_cnt
-cert_manager.acme_client.request.duration.sum,gauge,,,,The sum of the HTTP request latencies in seconds for the ACME client,1,cert-manager,cm.req_duration.sum
-cert_manager.acme_client.request.duration.count,gauge,,,,The count of the HTTP request latencies in seconds for the ACME client,1,cert-manager,cm.req_duration.cnt
-cert_manager.acme_client.request.duration.quantile,gauge,,,,The quantiles of the HTTP request latencies in seconds for the ACME client,1,cert-manager,cm.req_duration.quant
+cert_manager.http_acme_client.request.count,count,,,,The number of requests made by the ACME client,1,cert-manager,cm.acme_req_cnt
+cert_manager.http_acme_client.request.duration.sum,gauge,,,,The sum of the HTTP request latencies in seconds for the ACME client,1,cert-manager,cm.req_duration.sum
+cert_manager.http_acme_client.request.duration.count,gauge,,,,The count of the HTTP request latencies in seconds for the ACME client,1,cert-manager,cm.req_duration.cnt
+cert_manager.http_acme_client.request.duration.quantile,gauge,,,,The quantiles of the HTTP request latencies in seconds for the ACME client,1,cert-manager,cm.req_duration.quant
```

Comment:
This is a breaking change, when releasing it should be a major bump

Reply:
👍